app/views/case_logs/_log_list.html.erb

```diff
@@ -1,6 +1,9 @@
 <figure class="app-figure">
   <figcaption id="<%= title.dasherize %>" class="app-figure__caption">
-    <strong><%= pagy.count %></strong> total <%= title.downcase %>
+    <span class="govuk-!-margin-right-4">
+      <strong><%= pagy.count %></strong> total <%= title.downcase %>
+    </span>
+    <a class="govuk-link" download href="/logs.csv">Download (CSV)</a>
```

Comment:
We can also use the `download` attribute to give the downloaded CSV a filename… not sure what a useful name would be, though.

Reply:
Filename is set in the controller currently

Reply:
That’s probably best; `download` isn’t supported by IE11